client/my-sites/themes/themes-magic-search-card/index.jsx

```diff
@@ -65,6 +69,24 @@ const ThemesMagicSearchCard = React.createClass( {
 		this.setState( { searchIsOpen: false } );
 	},
 
+	searchTokens( input ) {
+		const tokens = input.split(/(\s+)/);
+		let cls;
```

Comment:
`cls` confused me initially. `className` might be better. Though maybe I'm just being slow this morning :)